### PR TITLE
fix Kafka consumer subscribe topics from pattern

### PR DESCRIPTION
**Pull request by amwyyyy**
fix Kafka consumer subscribe topics from pattern 
[#4865](https://github.com/apache/skywalking/issues/4865)

### CODE REVIEW

**Comment by wu-sheng** [comment]
Tests are failing, please check the logs and fix.

**Comment by wu-sheng** [comment]
Once all CIs passed, please add the plugin tests to cover this case.

**Comment by wu-sheng** [comment]
@amwyyyy Any update? Or are you clear what you should do?

**Comment by amwyyyy** [comment]
@wu-sheng I don’t know what to do, I’m groping

**Comment by wu-sheng** [comment]
When you are changing the plugin source codes, we would request you to add plugin tests. 
https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md

The point is this, we can't run your case locally, so we request you to add in the process of CI testing. Please read the document.

File: test/plugin/scenarios/kafka-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/kafka/controller/CaseController.java [hunk 1]
```diff
@@ -200,5 +208,44 @@ public void run() {
             consumer.close();
         }
     }
+
+    public class ConsumerThread2 extends Thread {
```
Comment (wu-sheng) [inline test/plugin/scenarios/kafka-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/kafka/controller/CaseController.java:212]: After you change this, there is an `expectedData.yaml` file, you should add the expected segment data in it.

**Comment by wu-sheng** [comment]
It seems tests fail. Could you try to run the tests locally?

**Comment by wu-sheng** [comment]
Merged, welcome and congrats.

**Pull request merged**